airbyte-integrations/connectors/destination-vectara/README.md

```diff
@@ -8,31 +8,12 @@ For information about how to use this connector within Airbyte, see [the documen
 ### Prerequisites
 **To iterate on this connector, make sure to complete this prerequisites section.**
 
-#### Minimum Python version required `= 3.7.0`
+#### Minimum Python version required `= 3.9`
```

Comment:
3.10 does not work with airbyte-CI, so setting minimum to 3.9 (same as source connectors using poetry)

Reply:
SG - I don't see a reason this won't work.